warehouse/ingest-scripts/src/main/resources/bin/ingest/shard-reindex.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
```

Comment:
A comment at the top denoting how to use this script with an example would be nice.